framework/src/main/java/org/apache/drill/test/framework/TestDriver.java

```diff
@@ -716,6 +716,11 @@ private void teardown() {
 		}
 	} 
 	connectionPool.releaseConnection(connection);
+
+    // Stop Apache Minio server if it was started for the S3 storage
+    if(cmdParam.excludeDependencies == null || !cmdParam.excludeDependencies.contains("s3")) {
```

Comment:
Hey my suggestion is to get these framework changes in separately in a different PR. That way we can keep tests and src independent.

Reply:
Ok. I'm closing this PR and will create 2 separate PR's for tests and logic.